openslides/core/views.py

```diff
@@ -353,3 +359,19 @@ def check_permissions(self, request):
         if (self.action in ('create', 'update', 'destroy') and
                 not request.user.has_perm('core.can_manage_tags')):
             self.permission_denied(request)
+
+
+class UrlPatternsView(utils_views.APIView):
+    """
+    Returns a dictonary with all url patterns as json.
+    """
+    URL_KWARGS_REGEX = re.compile(r'%\((\w*)\)s')
+
+    def get_context_data(self, **context):
+        result = {}
+        url_dict = get_resolver(None).reverse_dict
+        for url_pattern in filter(lambda key: isinstance(key, str), url_dict.keys()):
+            url = url_dict[url_pattern][0][0][0]
```

Comment:
Ah I think this won't work properly. The url_dict is a MultiValueDict that means one key contains possibly more than one value.
See [here](https://github.com/django/django/blob/master/django/core/urlresolvers.py#L446): `possibilities = self.reverse_dict.getlist(lookup_view)`
In your case: `possibilities = url_dict.getlist(url_pattern)`
So we need a more complex result.